resources/warnings.json

```diff
@@ -11263,7 +11263,7 @@
     "url": "https://www.jenkins.io/security/advisory/2022-03-15/#SECURITY-1351",
     "versions": [
       {
-        "lastVersion": "346.vd87693c5a_86c",
+        "lastVersion": "359.v35dcfdd0c20d",
         "pattern": ".*"
```

Comment:
```suggestion
        "lastVersion": " 356.va_90a_94ca_62ec",
        "pattern": "(0|34[236]|356)(|[.-].+)"
```

As above.

Reply:
Where is this process documented?  I'd like to prevent myself from making errors in the future.

Reply:
Documentation is rather minimal. It hasn't come up a lot, since maintainers can just inform us about fixes, rather than having to submit PRs.

There's Javadoc at https://github.com/jenkinsci/jenkins/blob/8eae71e5464f584533b7482a309ee903e65577f8/core/src/main/java/hudson/model/UpdateSite.java#L833-L843 in core. https://javadoc.jenkins.io/hudson/model/UpdateSite.WarningVersionRange.html is a bit awkward to read because `pattern` is private, so doesn't show, I should probably change that.

Besides that, there are all of the examples in other warnings in this file which give you an idea about what the expected contents are.